### ISSUE TITLE
Nom breaks `dbg` macro from std

### ISSUE THREAD

**Issue by chifflier** [Geal/nom#1022]
Nom defines a `dbg` macro, which unfortunately breaks the same macro from `std` (even without adding a `use` for it, it seems the namespaces aren't working as expected).

Minimal code to reproduce (nom 5.0, rustc 1.37):

```
#[macro_use]
extern crate nom;

fn test() {
    let a = 1;
    dbg!(&a);
}
```

fails with error:
```
error: unexpected end of macro invocation
 --> tests/nom.rs:7:12
  |
7 |     dbg!(&a);
  |            ^ missing tokens in macro arguments
```

Suggestion: rename `dbg`
Workaround:  replace line with `::std::dbg!(&a)`

**Comment by Geal** [comment]
hmm, I should have removed that one before, this makes a breaking change :/
I'll see if a 6.0 can happen soon enough

**Comment by Geal** [comment]
fixed in e799e76, `dbg!` is now `dbg_basic!`
